R/projpred.R

```diff
@@ -95,8 +62,13 @@ get_refmodel.brmsfit <- function(object, newdata = NULL, resp = NULL,
   # "additionally to `link`") like `refcat` and `thresholds` (see ?brmsfamily):
   if (!(isTRUE(dots$aug_data) && is_polytomous(family))) {
     family <- get(family$family, mode = "function")(link = family$link)
+  } else {
+    # TODO: uncomment the lines below as soon as the
+    # `extend_family_<family_name>` exist (in brms):
+    # family <- get(paste0("extend_family_", family$family, mode = "function"))(
+    #   family
+    # )
   }
-  family <- projpred::extend_family(family)
```

Comment:
so does that mean that extend family is no longer required to be called for the time being?

Reply:
Yes, it's not required. Actually, I don't know if it was ever required because I think the call to `extend_family()` in `init_refmodel()`  exists for quite a while. It doesn't harm to call `extend_family()` here in `get_refmodel.brmsfit()` (because `extend_family()` checks whether the family has already been extended) but I thought omitting this would be more straightforward.